src/lib/components/ratio/index.svelte

```diff
@@ -77,11 +79,15 @@
     gap: var(--result-layout-gap, var(--double-gutter));
```

Comment:
@dvdherron when at the smallest size, I wonder if we want to adjust or remove the gap on [data-layout='results']  since it is adding a double gap between the pass/fail section and the known issues section even though we have the contrast-defined section set to display none. 

![Screenshot 2023-12-18 at 10 03 21 AM](https://github.com/oddbird/oddcontrast/assets/1581694/04c1f9b0-de7b-4bc4-818e-ea7d1934827e)

Reply:
I made this smaller in https://github.com/oddbird/oddcontrast/pull/146/commits/72e099cb82200f7bf98680625eb19eb26f6c08bf

![image](https://github.com/oddbird/oddcontrast/assets/41588129/ba3465e7-89ab-4b8f-950f-4dbe453c434e)